src/keyframe-interpolations.js

```diff
@@ -25,7 +25,7 @@
         }).forEach(function(interpolation) {
           var offsetFraction = fraction - interpolation.startOffset;
           var localDuration = interpolation.endOffset - interpolation.startOffset;
-          var scaledLocalTime = localDuration == 0 ? 0 : interpolation.easing(offsetFraction / localDuration);
+          var scaledLocalTime = localDuration == 0 ? 0 : interpolation.easingFunction(offsetFraction / localDuration);
```

Comment:
Should the "property != 'easing'" test in the else clause below (line 33) now be "property != 'easingFunction'"? Or "property != 'easing' && property != 'easingFunction'"?